### ISSUE TITLE
python/meterpreter/reverse_http* doesn't attempt to get default proxy settings

### ISSUE THREAD

**Issue by Meatballs1** [Veil-Framework/Veil-Evasion#111]
:(

**Comment by ChrisTruncer** [comment]
Ah, I do believe you're correct.

I'll have to look into the code to get that added into it.  Thanks for bringing it up

**Comment by ChrisTruncer** [comment]
Were you able to verify this?  I started researching how to auto-detect proxy settings in Python, and I talked with @zerosteiner in #metasploit on freenode.  He helped explain that he believes (and it looks to be the case based off of the python meterpreter payload) urllib2 will auto-detect system proxy setting, unless they have been explicitly specified.  We're not specifying any proxy settings, and are using urllib2 to make the call to the handler, so it would seem that it SHOULD be detecting the proxy settings.  I personally don't have a lab environment at the moment to test this out, so I very well could be wrong.
